### PR TITLE
Adding examples for feed item sets.

### PR DESCRIPTION
**Pull request by mcloonan**
Change-Id: I18c2f8052a3dae826dc16753a9a5c070320fdb4f

### CODE REVIEW

File: examples/feeds/create_feed_item_set.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    # code that sets dynamic_location_set_filter. If your feed is an affiliate
+    # extension, set dynamic_affiliate_location_set_filter instead.
+
+		# 1) Location extension.
+    # set.dynamic_location_set_filter = client.resource.dynamic_location_set_filter do |filter|
+    #   filter.business_name_filter = client.resource.business_name_filter do |bnf|
+    #     bnf.business_name = "INSERT_YOUR_BUSINESS_NAME_HERE"
+    #     bnf.filter_type = :EXACT
+    #   end
+		# end
```
Comment (fiboknacky) [inline examples/feeds/create_feed_item_set.rb:47]: Some lines have strange indentation. Is this intended?

File: examples/feeds/create_feed_item_set.rb [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+		# 2) Affiliate extension.
```
Comment (fiboknacky) [inline examples/feeds/create_feed_item_set.rb:49]: The indentation looks strange?

File: examples/feeds/link_feed_item_set.rb [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Links the specified feed item set to the specified feed item. The specified
+# feed item set must not be created as a dynamic set, i.e. both
+# FeedItemSet::dynamic_location_set_filter and
```
Comment (fiboknacky) [inline examples/feeds/link_feed_item_set.rb:20]: `FeedItemSet::dynamic_location_set_filter` -> Is this the normal format of Ruby? If not, could you also change?

File: examples/feeds/create_feed_item_set.rb (hunk 1 above)
Comment (mcloonan) [reply examples/feeds/create_feed_item_set.rb:47]: Ugh, there are tabs intermixed with spaces. It looked fine in my IDE. Thanks for catching!

**Review by fiboknacky — APPROVED** [review]

**Review by xflesym — APPROVED** [review]

**Pull request merged**